src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -270,6 +278,20 @@ public Builder ()
 			BuildLogFile = "build.log";
 			Console.WriteLine ($"Using {XABuildExe}");
 			Console.WriteLine ($"Using {(RunningMSBuild ? "msbuild" : "xbuild")}");
+			var psi = new ProcessStartInfo ("msbuild") {
+				RedirectStandardOutput = true,
+				UseShellExecute = false,
+				CreateNoWindow = true,
+				WindowStyle = ProcessWindowStyle.Hidden,
+				WorkingDirectory = Root,
+				Arguments = $" {Path.Combine (Root, "FrameworkPath.targets")} /v:minimal /nologo",
+			};
+			if (IsUnix) {
+				using (var process = Process.Start (psi)) {
+					process.WaitForExit ();
+					frameworkSDKRoot = process.StandardOutput.ReadToEnd ().Trim ();
+				}
+			}
```

Comment:
Can we cache the result of this `msbuild` call so it doesn't run so many times throughout our tests? A `static` variable would probably work.